### PR TITLE
docs(grid, treelist): linked popup templates articles in customization

### PR DESCRIPTION
**Pull request by yanisslav**

### CODE REVIEW

File: components/grid/editing/popup.md [hunk 1]
```diff
@@ -260,6 +260,14 @@ The `GridPopupEditFormSettings` nested tag exposes the following parameters to a
 }
 ````
 
+### PopUp Form Customization
```
Comment (yordan-mitev) [inline components/grid/editing/popup.md:263]:
```suggestion
### Popup Form Customization
```

File: components/grid/editing/popup.md [hunk 2]
```diff
@@ -260,6 +260,14 @@ The `GridPopupEditFormSettings` nested tag exposes the following parameters to a
 }
 ````
```
Comment (yordan-mitev) [inline components/grid/editing/popup.md:262]: Could you please replace "PopUp" with "Popup" in the entire article?

File: components/grid/templates/popup-form-template.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ position: 50
 
 With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the Grid.
 
+>When a `FormTemplate` is declared, the Grid no longer handles the Update and Cancel operations.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:16]:
```suggestion
>When you declare a `FormTemplate`, the Grid no longer handles the Update and Cancel operations.
```

File: components/treelist/templates/popup-form-template.md [hunk 4]
```diff
@@ -13,6 +13,8 @@ position: 50
 
 With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the TreeList.
 
+>When a `FormTemplate` is declared, the TreeList no longer handles the Update and Cancel operations.
```
Comment (yordan-mitev) [inline components/treelist/templates/popup-form-template.md:16]:
```suggestion
>When you declare a `FormTemplate`, the TreeList no longer handles the Update and Cancel operations.
```

File: components/treelist/editing/popup.md [hunk 5]
```diff
@@ -445,6 +445,14 @@ The `TreeListPopupEditFormSettings` nested tag exposes the following parameters
 }
 ````
```
Comment (yordan-mitev) [inline components/treelist/editing/popup.md:447]: Please replace "PopUp" with "Popup" in the entire document.

**Comment by ntacheva** [comment]
- [ ] Add a TOC with links at the beginning of the Customization section - it is long and the last customization options are not easily visible.

**Review by ntacheva — CHANGES_REQUESTED** [review]
Listed above.

File: components/grid/templates/popup-form-template.md [hunk 6]
```diff
@@ -11,7 +11,11 @@ position: 50
 
 # Popup Form Template
 
-With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the Grid.
+With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the grid. To set it, provide contents to the `<FormTemplate>` inner tag of the `<GridPopupEditFormSettings>`.
+
+You can use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable. The context variable is of type `object` and can be cast to the model type to which the grid is bound.
```
Comment (ntacheva) [inline components/grid/templates/popup-form-template.md:16]:
```suggestion
You can use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable. The context variable is of type `object` and can be cast to the model type to which the Grid is bound.
```

File: components/treelist/templates/popup-form-template.md [hunk 7]
```diff
@@ -11,7 +11,11 @@ position: 50
 
 # Popup Form Template
 
-With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the TreeList.
+With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the TreeList. To set it, provide contents to the `<FormTemplate>` inner tag of the `<TreeListPopupEditFormSettings>`.
```
Comment (ntacheva) [inline components/treelist/templates/popup-form-template.md:14]:
```suggestion
With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the TreeList. To set it, provide contents to the `<FormTemplate>` inner tag of the `<TreeListPopupEditFormSettings>`.
```
```suggestion
With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the TreeList. Declare the desired custom content inside the `<FormTemplate>` inner tag of the `<TreeListPopupEditFormSettings>`.
```

**Review by ntacheva — APPROVED** [review]

**Pull request merged**